tasks/documentation.rb

```diff
@@ -1,4 +1,4 @@
-require_relative '../lib/shoulda/matchers/version'
+require 'shoulda/matchers/version'
```

Comment:
Why this change is done? Seems unrelated to the PR.

Reply:
Each change has been committed separately with a verbose commit message.
Please let me know if they're not clear enough.

Reply:
Nice commit message. I did not know that. Thanks! :+1: 

I'm just wondering, the project has 86 uses of require_relative, so why modify just this one?

If we intend to change several require_relative to require, I think it would be better to separate into another PR, but for now, I think we need to wait for more feedback on this change.

Reply:
Hehe, nice! 😄 
The rationale for each cop is documented here: https://docs.rubocop.org/rubocop-packaging/cops_packaging.html#require-relative-hardcoding-lib-rationale

So it's still preferred to use `require_relative` but just not cross-lib. More specifically, defintiely not from the test code to `lib/`.